npm-shrinkwrap.json

```diff
@@ -2,11 +2,36 @@
   "name": "@okta/okta-signin-widget",
   "version": "1.11.0",
   "dependencies": {
+    "@okta/ci-pkginfo": {
+      "version": "1.2.0"
+    },
+    "@okta/ci-update-package": {
```

Comment:
The [wiki](https://oktawiki.atlassian.net/wiki/display/eng/Sign-in+Widget+Development) said I should remove ci-
- ci-pkginfo
- ci-update-package
- install-with-shrinkwrap
- fs-events